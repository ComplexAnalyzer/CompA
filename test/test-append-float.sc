def main() {
    a = [];
    print(1.1?a); /* print 0 */
    a = a + [1.1];
    print(1.1?a); /* print 1 */
    func(a);
}


def func(a) {
    b = 1.2;
    a = a + [b];
    print(a[1]); /* prints 1.2 */
    return b;
}
