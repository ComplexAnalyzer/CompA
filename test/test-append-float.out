false
true
1.200000
